### ISSUE TITLE
i can't active tap dance

### ISSUE THREAD

**Issue by yulitri3** [NCKiser/3x3macropad#1]
![ss](https://user-images.githubusercontent.com/109904854/180852838-b80f4299-0b0e-490c-ae5d-62f80eaa8d4c.png)
I get a problem when I activate the tap on rule.mk, can you help me

**Comment by NCKiser** [comment]
What files are you changing, the root rules.mk, or the rules.mk within the vial keymap folder?
Tap dance would need to be enabled in both rules.mk files

**Comment by yulitri3** [comment]
i just change maps pin pro micro on config.c . in the rules.mk file in the keymap folder there is no "TAP DANCE ENABLE", do I have to add it if I set "TAP_DANCE_ENABLE" in the root rules.mk = yes as in the root rules.mk?

**Comment by NCKiser** [comment]
Yes, you will need to add "TAP_DANCE_ENABLE = yes", or equivalent, to both rules.mk.

**Comment by yulitri3** [comment]
I have done it and the result is the same, but the compilation is successful if I set tap_dance in the root rules.mk to no, and the result is that in the vial application there is no tap dance combo qmk setting, there are only keymaps and macros. i have tried your .hex file and it works but i can't find pin D6 on my pro micro can you help me to create .hex file with pin

ROW F4, F5, F7
COL D1, D0, D4
